modules/openy_features/openy_prgf/openy_prgf.install

```diff
@@ -32,3 +45,21 @@ function openy_prgf_update_8002() {
     'fields_unset_paragraphs.field_bottom_content'
   );
 }
+
+/**
+ * Enable webforms module.
+ */
+function openy_prgf_update_8003() {
+  $config_dir = drupal_get_path('module', 'openy_prgf') . '/config/install/';
+  // Import new configuration
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'core.entity_form_display.paragraph.webform.default.yml',
+    'core.entity_view_display.paragraph.webform.default.yml',
+    'core.entity_view_display.paragraph.webform.default.yml',
+    'field.field.paragraph.webform.field_webform_webform.yml',
+    'field.storage.paragraph.field_webform_webform.yml',
+    'paragraphs.paragraphs_type.webform.yml',
+  ]);
+}
```

Comment:
This update should live in paragraph module. See this as example https://github.com/ymcatwincities/openy/tree/8.x-1.x/modules/openy_features/openy_prgf/modules